### ISSUE TITLE
Multiple Identities for the same network cause the service to crash when stopping "all the services"

### ISSUE THREAD

**Issue by JeremyTellier** [openziti/desktop-edge-win#98]
![image](https://user-images.githubusercontent.com/56126806/89849366-5badab00-db56-11ea-978c-13490fd0e28b.png)

Clicking "STOP" button causes "End of Stream" error in the UI

**Comment by dovholuknf** [comment]
steps to reproduce:
* make 3 identities for the same network (might need to be from demo4)
* enroll with tunneler
* click stop

observe service crash and output to stdout/err: 
```
panic: listen udp6 [::1]:53: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

goroutine 24 [running]:
github.com/openziti/desktop-edge-win/service/cziti.runListener(0xc00009caa8, 0x10, 0x10, 0x35, 0xc000086420)
        c:/git/github/openziti/desktop-edge-win/service/cziti/dns-proxy.go:145 +0x307
created by github.com/openziti/desktop-edge-win/service/cziti.runDNSserver
        c:/git/github/openziti/desktop-edge-win/service/cziti/dns-proxy.go:120 +0x107
```

**Comment by dovholuknf** [comment]
Second attempt at reproducing crashes the desktop edge for windows - but no actual output is shown on stdout/err nor in the logs. process 'just exits'

**Comment by dovholuknf** [comment]
resolved around 1.2.14+
